RWrapper/mspc_1.0.1/DESCRIPTION

```diff
@@ -0,0 +1,19 @@
+Package: mspc
+Title: MSPC program
+Version: 1.0.0
+Authors@R: 
+    person(given = "First",
+           family = "Last",
+           role = c("aut", "cre"),
+           email = "first.last@example.com",
+           comment = c(ORCID = "YOUR-ORCID-ID"))
+Imports: processx, BiocManager, rtracklayer, stats, tools, methods
+Description: Runs the mspc program. MSPC comparatively evaluates ChIP-seq peaks and combines the statistical significance of repeated evidences, with the aim of lowering the minimum significance required to “rescue” weak peaks; hence reducing false-negatives.
+License: GPL-3
+Encoding: UTF-8
+LazyData: true
+RoxygenNote: 7.1.1
+NeedsCompilation: no
+Packaged: 2021-05-04 03:45:20 UTC; Meriem
+Author: First Last [aut, cre] (YOUR-ORCID-ID)
+Maintainer: First Last <first.last@example.com>
```

Comment:
I think Meriem should be the maintainer for now, then in the future I can become the maintainer if needed

Reply:
Alright, will do!